create-test-tables.sh

```diff
@@ -0,0 +1,32 @@
+curl -v 'http://localhost:8124/' --data-binary @- << EOF
+  CREATE TABLE patient(
+    id String NULL,
```

Comment:
Might be better to choose either snake or camel case for the attribute names. There seems to be a tendency towards writing identifiers in snake case in SQL.

Reply:
Unfortunately this has to conform to the identifiers used in the default `fhir-mapping.json` file. I'm not sure it's the scope of this PR to change that default mapping as well.